vembrane/__init__.py

```diff
@@ -23,45 +23,93 @@
 }
 
 
-def filter_vcf(vcf: VariantFile, expression: str) -> Iterator[VariantRecord]:
+def get_annotation_keys(header):
+    for rec in header.records:
+        if rec.get("ID") == "ANN":
+            return list(map(str.strip, rec.get("Description").split("'")[1].split("|")))
+    return []
+
+
+def parse_annotation_entry(entry: str):
+    return list(map(str.strip, entry.split("|")))
+
+
+def filter_annotation_entries(entries: list, ann_filter_expression: str):
+    for entry in entries:
+        env = dict(zip(annotation_keys, parse_annotation_entry(entry)))
```

Comment:
`annotation_keys` is not defined here; either move this def into the filter_vcf definition or make annotation_keys a param (or a global...)

Reply:
good catch.